source/MaterialXShaderGen/ShaderGenerators/Glsl/GlslShaderGenerator.h

```diff
@@ -42,6 +42,10 @@ Uniform variables :
     u_frame                             float   The current frame number as defined by the host application
     u_time                              float   The current time in seconds
     u_geomattr_<name>                   <type>  A named attribute of given <type> where <name> is the name of the variable on the geometry
+    u_numActiveLightSources             int     The number of active light sources
```

Comment:
Might want to mention this is clamped by the max # of lights defined on the generator.

Reply:
Good point, will add that in.